apps/io-web-profile/.opex/external/env/prod/config.yaml

```
@@ -0,0 +1,16 @@
+oa3_spec: apps/io-web-profile/api/external.yaml
+name: opex_io_web-profile_api_external
```

if I'm not mistaken a name like this would result in minimal or no changes to plan result and would not require a resource deletion since identical dashboard and alerts are already present. let me know what you think
```suggestion
name: opex_iowebprofilebackend_external
```

---

[b0fe571](https://github.com/pagopa/io-auth-n-identity-domain/pull/156/commits/b0fe571a7d8e8f428ca154954a917db66c3708be)

Also changed the terraform state, now we use the same one as the previous repo